main.go

```diff
@@ -156,6 +169,26 @@ func main() {
 			handle.LoadRegionLablesAndGC(dumpFileRegionLabelRule, dumpfilePath, ctx, []string{*pdAddr}, isCanRun, *isSkipConfirm)
 
 		}
+	case opDumpEnabledKS:
+		{
+			dumpfile, err := os.OpenFile(*dumpFilepath, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0666)
```

Comment:
dumpFilepath is used to store archived keyspace list, so we can set another dump file path to store the enabled keyspace